src/Routers/UsersRouter.js

```diff
@@ -42,8 +42,9 @@ export class UsersRouter extends ClassesRouter {
     if (!req.info || !req.info.sessionToken) {
```

Comment:
req.info ?
What is this ? 
On Parse i used 'X-Parse-Session-Token' in headers, how it work now ?
Thanks